clang/test/CodeGenSYCL/simplifycfg.cpp

```diff
@@ -0,0 +1,35 @@
+// RUN: %clangxx -fsycl -fsycl-device-only %s -O3 -S -o - | FileCheck %s
```

Comment:
Should this be in `sycl/test` instead? We don't usually have tests requiring `clangxx `driver in FE.

Reply:
> Should this be in `sycl/test` instead? We don't usually have tests requiring `clangxx `driver in FE.

No, but we should invoke FE directly (i.e. %clang_cc1).

Reply:
%clang_cc1 doesn't recognize -fsycl argument.

Reply:
It has no such option. You may use something like:
`%clang_cc1 -fsycl-is-device -triple spir64-unknown-unknown -flegacy-pass-manager -O3 -emit-llvm %s -o -`

Reply:
It turned out that with %clang_cc1 SimplifyCFG still break code and test doesn't pass. Investigating.

Reply:
I suppose that also '-mllvm -sycl-opt' should be added into command line for %clang_cc1.

Reply:
Found the reason - missed `-sycl-opt` argument.